modyn/common/trigger_sample/trigger_sample_storage.py

```diff
@@ -232,6 +232,10 @@ def _parse_file(self, file_path: Path) -> np.ndarray:
         Returns:
             list[tuple[int, float]]: List of trigger samples.
         """
+
+        if not os.path.isfile(file_path):
+            return np.ndarray(0, dtype="i8,f8")
```

Comment:
When there are few samples, it may happen that some workers don't have any samples to store. Therefore, they do not write anything; you get an error if you try to read their file. This way, it returns an empty array if the worker has not written anything.

Reply:
Sure